hello-slack/prd.md

```
@@ -0,0 +1,301 @@
+
+⸻
+
+:pushpin: PRD（プロダクト要件定義）
+
+:white_check_mark: 目的
+	•	Slack上でメンションされたとき、ボットが応答する。
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)